nessie/src/main/java/org/apache/iceberg/nessie/NessieViewOperations.java

```diff
@@ -0,0 +1,157 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iceberg.nessie;
+
+import java.util.Map;
+import org.apache.iceberg.exceptions.AlreadyExistsException;
+import org.apache.iceberg.exceptions.NoSuchViewException;
+import org.apache.iceberg.io.FileIO;
+import org.apache.iceberg.view.BaseViewOperations;
+import org.apache.iceberg.view.ViewMetadata;
+import org.apache.iceberg.view.ViewMetadataParser;
+import org.projectnessie.client.http.HttpClientException;
+import org.projectnessie.error.NessieConflictException;
+import org.projectnessie.error.NessieNotFoundException;
+import org.projectnessie.model.Content;
+import org.projectnessie.model.ContentKey;
+import org.projectnessie.model.IcebergTable;
+import org.projectnessie.model.IcebergView;
+import org.projectnessie.model.Reference;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class NessieViewOperations extends BaseViewOperations {
```

Comment:
Generally, `NessieViewOperations` and `NessieTableOperations` share a lot of similar code - critical code duplicated.
I'd prefer to move the common parts to a package-private interface with default implementations.

Reply:
I would suggest to do such bigger refactorings as part of a separate PR, otherwise it will be hard to review

Reply:
The critical code is already extracted to `NessieIcebergClient`. 

`NessieTableOperations` and `NessieViewOperations` has only two methods `doRefresh` and `doCommit` which operates on different objects, one is `TableMetadata` and one is `ViewMetadata`.

Reply:
I checked again. `NessieTableOperations` and `NessieViewOperations` is based on two different classes `BaseMetastoreTableOperations` and `BaseViewOperations`. I cannot make a base class extending these two. 

Also, having a common interface is not much useful here as both view and table cannot implement the same interface. 

Lastly checked about a static util class for common functionalities. The code looks similar but it is lot different as one use view and one use table (So one uses table metadata and other uses view metadata).

Reply:
I don't understand the reason that two classes cant implement the same interface???